### PR TITLE
Halts another macro suicide method

### PR DESCRIPTION
**Pull request by SamHPurp**
Adding a cooldown to burning yourself with a lightbulb. All interactable things should already have cooldowns apply, this is an outlier.

Also this has balance implications due to your ability to kill yourself as fast as if you typed `suicide`, but by using this method you are eligible for mid-round antags.

### CODE REVIEW

**Comment by LightFire53** [comment]
This is lacking in detail what it actually changes. 
If I'm am reading the code right (which I am probably not), you are adding a cooldown to grabbing a light bulb while you don't have gloves on?

**Comment by Spartan6** [comment]
Pretty much, I think spam clicking a bulb kills you.

**Comment by SamHPurp** [comment]
@LightFire53 in a word: Yes.

**Comment by KingPhilipIII** [comment]
I could easily achieve the same results by asking the Chaplain to incinerate me, the chef to gib me, throwing myself out an airlock, smashing my own face in, running into the gateway(Sometimes) or any myriad of ways to kill myself.

I'm not inherently against the PR, because I don't really care all that much, but I think the justification of preventing exploitation is invalid.

**Comment by Citinited** [comment]
@KingPhilipIII all of the things you listead are a) not near-instantaneous, b) cannot be done trivially or in the brig. The issue here is that spamming lightbulbs is too easy to keep.

**Comment by KingPhilipIII** [comment]
@Citinited Acknowledged.

**Comment by Kyep** [comment]
If someone wants to suicide in the brig, whilst still being eligible for midround antags, why don't they just punch themselves to death?

**Comment by alexkar598** [comment]
Probably because they wil go into crit for 5 minutes and by then the sentence would be done anyways

**Comment by Birdtalon** [comment]
I didn't even know this was a thing.

**Comment by taukausanake** [comment]
I don't think it's justified in saying this is as fast as the Suicide verb whenever or not it is. That's just pulling something out from somewhere else to to try and justify it.
That being said however, I see reason for most things it not all to have some form of cooldown. No one should be able to repeatedly grab a hot light bulb and die from it, not unless it's an actual Suicide verb

**Comment by SamHPurp** [comment]
@taukausanake when I tried this from start to finish I was speaking to the ghosts in sub-10 seconds.

**Comment by Bigtimetripod** [comment]
#4246

#4367

Light bulbs are the new antags once agains. Cannot really say "i ded, plox nerf", but why the hell would ye kill yerself with a lightbulb... Just turn yer oxygen tank pressure release to 0 and smash yerself with the same darn oxygen bottle...

Yesyes, if ye die "normally" ye can come back as a midround antag/anything else, if ye suicide, ye cannot.

**Comment by The-Driker** [comment]
An interesting example of this happened tonight. A prisoner managed to husk themselves in solitary confinement using the light-bulb. This was done in seconds after getting their cuffs off.

**Comment by Necaladun** [comment]
Arguments for and against suicide, husking, etc, notwithstanding, I don't see any reason not to have a cooldown on something like this. In general, shit should have cooldowns on it.

**Comment by SamHPurp** [comment]
20 days on, can this just be merged now?

File: code/modules/power/lighting.dm [hunk 1]
```diff
@@ -454,6 +454,7 @@
 			to_chat(user, "You telekinetically remove the light [fitting].")
 		else
 			if(user.a_intent == INTENT_DISARM || user.a_intent == INTENT_GRAB)
+				user.changeNext_move(CLICK_CD_MELEE)
```
Comment (Fox-McCloud) [inline code/modules/power/lighting.dm]: This should be at line 431, not here.

**Comment by SamHPurp** [comment]
Whatever, updated.

**Comment by tigercat2000** [comment]
This PR is getting merged, but the Heads and Maints would like it to be known that we are not looking to stop suicide methods altogether or make suicide harder at this point; this is merely in order to standardize cooldowns across the board. We're never going to be able to stop every method of killing yourself without becoming unable to rejoin the round, and we shouldn't try.

**Pull request merged**